packages/product/src/scripts/migration-up.ts

```diff
@@ -33,11 +33,15 @@ export async function runMigrations({
 
   try {
     const migrator = orm.getMigrator()
-    await migrator.up()
+    const pendingMigrations = await migrator.getPendingMigrations()
+    logger.info("pendingMigrations", pendingMigrations)
```

Comment:
```suggestion
```

Reply:
This is to show which migrations will be ran, do you think it is not needed?

Reply:
In that same, I would do: `"Running pending migrations: "`

Reply:
That is ok for me 👍